### ISSUE TITLE
Cinch & Teardown Docs Could Use an Update

### ISSUE THREAD

**Issue by jaypoulz** [RedHatQE/cinch#195]
# Background
The default workflow when you pass it an inventory file is to run the ansible-playbook site.yml. Because cinch wraps around the ansible-playbook call, it isn't possible to pass dynamic arguments to cinch. This becomes an issue when you have a playbook that needs dynamic inputs.

# Example
Inventory
```
rhel_base_urls:
  x86_64: "http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server"
  ppc64le: "http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server"
  s390x: "http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/system-z/7/7Server"
  aarch64: "http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/arm/7/7Server"
rhel_base: "{{ rhel_base_urls[arch] }}"
```

Here I am trying to pass along an arch variable to specify which URL to use for the RHEL-base.
When I manually call `ansible-playbook` I can override this variable with some JSON such as: `{ arch: 'x86_64' }`. However, when I call cinch, this file needs to be static, or it needs to pull from the environment variables.

I think it's pretty clear from this example that it would be strange to have an environment variable $ARCH on my x86 machine that is something other than x86.

Furthermore, using environment variables in a run like this poses security risks if the data being passed includes sensitive passwords.

# Update
It turns out that this and many other ansible commands are actually supported by cinch. This should be reflected in the online docs, the --help command line, and potentially in a man page if necessary.

**Comment by jaypoulz** [comment]
https://github.com/RedHatQE/cinch/blob/master/cinch/bin/entry_point.py line 18 Seems to implement this issue..... verifying now.

This works as expected. 👍 

That being said, having to go to the source to figure out this usage pattern is not ideal.

**Comment by robled** [comment]
In any case, we should make this feature more clear in the docs, so I suggest we leave this ticket open as a reminder to improve the CLI help for the 'cinch' command as well as the sphinx docs.

**Comment by robled** [comment]
This doc update will also include more info about how to modify cinch variables without having to modify cinch itself.

**Comment by robled** [comment]
This has been completed and released as part of cinch 1.0.0:

https://github.com/RedHatQE/cinch/releases/tag/v1.0.0
